package-lock.json

```diff
@@ -284,7 +284,7 @@
                 "yargs-parser": "^13.1.2"
             },
             "engines": {
-                "vscode": "1.58.0-insider"
+                "vscode": "1.57.0-insider"
```

Comment:
Don't we want this on 1.58 still?

Reply:
I have an issue for that (https://github.com/microsoft/vscode-jupyter/issues/6195)
If I change it now that it doesn't exist, all tests fail

Reply:
Ok that works. I was just confused as this was the merge from release branch. Did you manually change this?